Documentation/Signoffs/php-communication-signoff.md

```diff
@@ -0,0 +1,82 @@
+
+![TransmitterCommunication](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/142913669/cfb66cac-4edb-49bc-8843-ddd06d3072f7)
+
+![RecieverCommunication](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/142913669/8ede150c-271b-4bfd-8071-dab157138e04)
+
+
+## Constraints
+| No. | Constraints                                                                                    | Origin                              |
+| --- | ---------------------------------------------------------------------------------------------- | ----------------------------------- |
+|  1  | System shall not require an internet connection to work and communicate with the head unit and sensors | Home Owners, Insurance Agencies, & Team Supervisor |
+|  2  | At least one sensor shall be placed within 200ft of the head unit. | Project Team & Broader Implications & Standards |
+|  3  | Sensors shall be within 32ft from each other for every sensor outside of 200ft of the head unit. | Project Team & Broader Implications & Standards |
+|  4  | Sensor communication shall send the sensor name, number, data type, and raw data to the head unit. | Project Team |
+|  5  | Sensor modules shall not send more than 64 bits (1 packet) of data per data request when transmitting information to the head unit. | Project Team & Standards|
+|  6  | The head unit shall, at minimum, receive data every 24 hours, when the data from the sensor reaches the specific limit for each sensor, or when the user requests the data. | Project Team |
+
+<sup>1</sup> In order for the system to not require an internet connection, the system will be designed to work on a Personal Area Network (PAN). In order to achieve this, the Zigbee communication protocol will be used.
+
+<sup>2</sup> In order for the system to work properly all of the sensors have to be able to communicate with the head unit. At least one sensor has to be within 200ft of the head unit because the XB24's maximum range allows for 200ft of indoor communication. Indoor means through the interior walls of a house.
+
+<sup>3</sup> The sensors have to be placed within 32ft of each other for the minimum transmission and reception range of the Zigbee-compliant devices.
+
+<sup>4</sup> The head unit needs to know the sensor information to be able to organize the data of each sensor. Knowing this allows the information to be more easily displayed to the user.
+
+<sup>5</sup> This is the maximum amount of data that Zigbee can send per data request from the head unit. This is also necessary to preserve storage and power consumption. It will also prevent overflow and loss of information.
```

Comment:
What does this mean? Why is it a constraint?

Reply:
We reworded this constraint to make it related to amount of data sent and the speed.